### ISSUE TITLE
Don't link sfml-main to GUI examples

### ISSUE THREAD

**Issue by binary1248** [LaurentGomila/SFML#610]
Time and again, people post on the SFGUI forum and the SFML forum regarding "stuff that doesn't work" or some corrupted graphics etc. When I ask them if their drivers/system is set up properly, they tend to say "the SFML examples work fine" and assume that everything is good. When I tell them to enable the console they find out that SFML has been spamming them with OpenGL errors and what not but they just didn't have the console enabled to see them. Had they seen those errors when running the examples to begin with, I think they would have fixed what was wrong with their system before assuming that something is wrong with the libraries.

Is it a good idea to make the examples display a console in the background? It would be easier and faster for people to diagnose problems that way before starting their own project with a broken environment.

I initially thought about only enabling the console subsystem for the debug build of the examples but CMake doesn't allow setting WIN32_EXECUTABLE based on build-type in multi-configuration mode and the only workaround has been broken for the last 2 years, [see here](http://public.kitware.com/Bug/view.php?id=12566). So if we went ahead and did this, it is all or nothing.

**Comment by eXpl0it3r** [comment]
Superseded by #766
